Firestore/tests/Snippet/FieldValueTest.php

```diff
@@ -65,7 +65,7 @@ public function testDeleteField()
                     ],
                 ],
             ],
-        ])->willReturn([[]]);
+        ])->willReturn([[]])->shouldBeCalledTimes(1);
```

Comment:
Changes in this file help suppress the test warnings for risky tests: `This test did not perform any assertions`